packages/editor-ui/src/views/SettingsSso.test.ts

```diff
@@ -0,0 +1,203 @@
+import { createTestingPinia } from '@pinia/testing';
+import { createComponentRenderer } from '@/__tests__/render';
+import SettingsSso from './SettingsSso.vue';
+import { useSSOStore } from '@/stores/sso.store';
+import { useUIStore } from '@/stores/ui.store';
+import { within, waitFor } from '@testing-library/vue';
+import userEvent from '@testing-library/user-event';
+import { mockedStore } from '@/__tests__/utils';
+
+const renderView = createComponentRenderer(SettingsSso);
```

Comment:
if i implement you suggestion, the tests break because the state of the pinia stores is shared.

![image](https://github.com/user-attachments/assets/ead4984c-b6fa-4819-8fff-f56ffd1576ea)

it's easier to create a fresh instance of the store each time, so every test is isolated from the others

Reply:
Ah yes, you're right
Maybe we should change `createComponentRenderer` so it can use `createTestingPinia` if no other option is passed and call it each time when rendering a component.
It's not very likely anyways I want to share store state between the tests

Reply:
But that for another time